velox/exec/SortBuffer.cpp

```diff
@@ -285,11 +285,13 @@ void SortBuffer::spillOutput() {
 }
 
 void SortBuffer::prepareOutput(uint32_t maxOutputRows) {
-  VELOX_CHECK_GT(maxOutputRows, 0);
+  VELOX_CHECK_GT(static_cast<vector_size_t>(maxOutputRows), 0);
```

Comment:
This will overflow

Reply:
Check fails on overflow. Or we need to add another check
VELOX_CHECK_LE(maxOutputRows, 0x7fffffff);